### PR TITLE
fix(CreateChatView): Move `ActivityCenterPopup` to appmain

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #6345

### What does the PR do

Move `ActivityCenterPopup` to `AppMain` and add creation method to `Global`

### Affected areas

ActivityCenter

### CODE REVIEW

File: ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml [hunk 1]
```diff
@@ -8,12 +8,16 @@ import StatusQ.Controls 0.1
...
 import shared.popups 1.0
+import shared.views.chat 1.0
+
 import utils 1.0
 
 import "../views"
 import "../panels"
 
 Popup {
+    id: activityCenter
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml:19]: "root" :)

File: ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml [hunk 2]
```diff
@@ -28,11 +32,14 @@ Popup {
     property bool hideReadNotifications: false
     property var store
     property var chatSectionModule
-    property var messageContextMenu
+    property var messageContextMenu: MessageContextMenuView {
+        id: contextmenu
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml:36]: need this `id`?

File: ui/app/mainui/AppMain.qml [hunk 3]
```diff
@@ -762,6 +768,20 @@ Item {
             }
         }
 
+        Component {
+            id: activityCenterPopupComponent
+            ActivityCenterPopup {
+                id: activityCenter
+                height: appView.height - 56 * 2 // TODO get screen size // Taken from old code top bar height was fixed there to 56
```
Comment (caybro) [inline ui/app/mainui/AppMain.qml:775]: Time to fix this now that we are in AppMain? ;)

**Review by caybro — COMMENTED** [review]
Some random blurbs inline :)

File: ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml (hunk 1 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml:19]: no pls, it takes more changes for task which is not refactoring :)

File: ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml (hunk 2 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml:36]: I think no... will fix :)

File: ui/app/mainui/AppMain.qml (hunk 3 above)
Comment (borismelnik) [reply ui/app/mainui/AppMain.qml:775]: okay, agreed :D

File: ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml (hunk 2 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml:36]: I think no... will fix :)

File: ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml (hunk 1 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml:19]: no pls, it takes more changes for task which is not refactoring :)

Comment (caybro) [reply ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml:19]: Ok, fair enough

**Comment by glitchminer** [comment]
Clicking on the activity center icon in a new chat now opens the activity center.
image

**Comment by PascalPrecht** [comment]
The tooltip for the activity center button seems to be off again.
@anastasiyaig this a known issue?

**Comment by anastasiyaig** [comment]
@PascalPrecht  not sure, probably something very recent that was not logged. At least i cant see it across the issues list on GH

**Comment by PascalPrecht** [comment]
I'm sure we've fixed this like 300 times.

**Comment by glitchminer** [comment]
@PascalPrecht, @anastasiyaig, tooltip alignment raised as new issue. https://github.com/status-im/status-desktop/issues/6700

**Review by glitchminer — APPROVED** [review]
Tested and approved.

**Review by stefandunca — APPROVED** [review]

**Pull request merged**